### DISCUSSION TITLE
How do you change algorithm? It shows me a card after 1 day. But if I get it wrong, I wan to see it in 1m，just like the anki there are four options，the first can be seen in 1 min or 6 min

### DISCUSSION THREAD

**Discussion by SupergeniusLZY** [st3v3nmw/obsidian-spaced-repetition#808] [Q&A]
How do you change the obsidian spaced repetition algorithm? It shows me a card after 1 day. But if I get it wrong, I want to see it in 1m, just like the anki there are four options， the first option can be seen in 1 min or 6 minute , if it can custom algorithm like another obsidian plugin _recall_, it will be much better. Anticipating your improvement,I'd appreciate any information you could give me. thx!

**Comment by tomasmaillo** [comment]
That feature is already in the works at #569, #503. In theory, should be getting implemented soon! 

If you cant wait, you can use [BRAT](https://github.com/TfTHacker/obsidian42-brat) to install the beta for the plugin. You can find the link for it on #569

**Reply by SupergeniusLZY** [reply]
thanks a lot！bro
